tests/MongoDB.Driver.Core.Tests/Core/Authentication/ScramSha256AuthenticatorTests.cs

```diff
@@ -60,31 +61,36 @@ public class ScramSha256AuthenticatorTests
         private static readonly string _clientRequest2 =
             $"c=biws,r={_clientNonce}{_serverNonce},p=dHzbZapWIk4jUhN+Ute9ytag9zjfMHgsqmmiz7AndVQ=";
         private static readonly string _serverReponse2 = "v=6rriTRBi23WpRR/wtup+mMhUZUn/dB5nLTJRsjl95G4=";
-        
+
         private static void Authenticate(
             ScramSha256Authenticator authenticator,
             IConnection connection,
-            ConnectionDescription description,
             bool async)
         {
             if (async)
             {
-                authenticator.AuthenticateAsync(connection, __description, CancellationToken.None).GetAwaiter().GetResult();
+                authenticator
```

Comment:
Idk why this method is not used at all. Also, it looks like it was implemented not fully correct. But I think it's a good test method, so I have used it

Reply:
Sounds good.